### PR TITLE
Revert "add top level software keys (#42)"

### PR DESCRIPTION
**Pull request by getvictor**
This reverts commit 5e21dc38944acc85879e38255c01c55d2896fc9f.

### CODE REVIEW

**Comment by getvictor** [comment]
Don't merge if https://github.com/fleetdm/fleet/pull/20847 was already merged.

**Review by noahtalerman — APPROVED** [review]

**Comment by lucasmrod** [comment]
Ah, @noahtalerman see Victor's last comment, we don't need this as we will be merging https://github.com/fleetdm/fleet/pull/20847 (pending review by Luke).

**Comment by lucasmrod** [comment]
But no worries, I'll create a separate PR to revert the revert.

**Pull request merged**
